### ISSUE TITLE
Users are unable to access to the segments present in Venn Chart using voice access in windows.

### ISSUE THREAD

**Issue by msftedad** [highcharts/highcharts#24653]
**Test Environment:**
OS: Windows 11 Version 25H2 (OS Build 26200.8390)
Browsers: Edge (Version 148.0.3967.70), Chrome (Version 148.0.7778.168)
Tool: Voice Access
URL: https://www.highcharts.com/demo/highcharts/venn-diagram

**User Impact:** 
Users with motor disability will not be able to interact with the control if on giving command 'Show numbers', Voice control is not showing numbers for the segments present in Venn chart due to which users will find it difficulty accessing the Venn Chart.

**Repro Steps:**

1. Open the URL: https://www.highcharts.com/demo/highcharts/venn-diagram in edge/ chrome browser.
2. Venn Diagram Demo page will open.
3. Give command as 'Show Number' to access the segment present in Venn chart.
4. Verify whether users are unable to access to the segments present in Venn Chart using voice access or not.

**Actual Result:**
Users are unable to access to the segments present in Venn Chart using voice access in windows.

**Refer Attachment:**
1. Users are unable to access to the segments present in Venn Chart using voice access in windows.png
2. Users are unable to access to the segments present in Venn Chart using voice access in windows.mp4

https://github.com/user-attachments/assets/519af385-1eba-40ab-b6ce-56f0e6441896
<img width="1919" height="1138" alt="Image" src="https://github.com/user-attachments/assets/206dfcc9-9f69-4e8c-99ec-76c6afb84376" />

**Expected Result:**
Users should be able to access to the segments present in Venn Chart using voice access in windows.

**Comment by magdalena-gut** [comment]
Hi @msftedad, thank you for bringing this to our attention! This appears to be a general limitation affecting all chart types, not just Venn. I've added it to our backlog for further work.
